### DISCUSSION TITLE
2023-04-26 - Weekly Hosting Meeting

### DISCUSSION THREAD

**Discussion by Crixu** [WordPress/hosting-handbook#144] [Weekly Meetings]
# Meeting

/here < meeting type="hosting" > Who’s here to chat about hosting?

👋 Hello! This is the weekly meeting for the WordPress.org Hosting Team, who works with hosts and developers to help improve the WordPress user experience. This is a great time to ask any questions you may have, or visit Make/Hosting for more info. 

🥇 If you participate in the Five for the Future program, drop your WordPress.org username in a thread here 👉

🤗 If you're new here, welcome! You are welcome to introduce yourself if you like in a thread here 👉

-----

## Agenda

📓 ··· Here is the agenda for this meeting ···

**WordPress Community**
- WordPress Notifications
- More Effective Contributor Days

**Hosting Team Time**
- SQLite Database Update
- WP CLI Update
- Contributor Days

**Hosting Team Projects**
- Advanced Administration Handbook
- Host Test Runners

**Open Floor**
- Anything not in the agenda

Would anyone like to add to or change any items on the agenda?

OK, let’s get started with the WordPress Community news.

-----

### :wordpress: ··· WordPress Notifications  ··· 

[WP Feature Notifications: 2023 Status Update](https://make.wordpress.org/core/2023/04/25/wp-feature-notifications-2023-status-update/)

The [WP Feature Notifications](https://github.com/WordPress/wp-feature-notifications) (formerly WP Notify) project was[ first proposed in 2019](https://make.wordpress.org/core/2019/08/05/feature-project-proposal-wp-notify/) in order to improve and modernise the way in which WordPress handles user notifications.

In the last few months we’ve had a good amount of progress, as well as a number of changes, which have led to the recent publication of [prerelease 0.2.0](https://github.com/WordPress/wp-feature-notifications/releases).

### :community: More Effective Contributor Days

[Cultivating More Effective Contributing on Contributor Days](https://make.wordpress.org/community/2023/04/19/cultivating-more-effective-contributing-on-contributor-days/)

Contributor Days are usually single day events focused on all things related to contributing to WordPress.

However, at many events the overwhelming majority of time at contributor day is usually spent in one of two areas:
- Helping people decide which team’s mission and work speaks to them and fits their skill sets or passions
- Onboarding people interested in specific teams in order to get started contributing

The post is not only meant to document the discussion and several action items that were identified, but also to gather any feedback from the wider community and volunteers with an interest in helping to complete the related initiatives.

-----

### :wordpress: ··· SQLite Update ··· 

[[Status update on the SQLite project](https://make.wordpress.org/core/2023/04/19/status-update-on-the-sqlite-project/)]

The initial implementation was included in the [Performance Lab](https://w.org/plugins/performance-lab) plugin and then released as a [stand-alone plugin](https://w.org/plugins/sqlite-database-integration/).

Why should this be in Core and not as a plugin?

In its current form, the SQLite implementation is a plugin. Just like all plugins, it can only be installed on a pre-existing website. As a result, a site can only use an SQLite database if it already has a MySQL database – effectively negating all the benefits that SQLite can bring to WordPress.

Using the featured plugin is a great way to allow users to test the implementation and iron out any issues etc. However, long-term, it doesn’t make sense to use it as a plugin.

The proof-of-concept UI in that implementation checks if the server supports SQLite and MySQL:

- If SQLite is supported but not MySQL, then it defaults to SQLite and no UI option is shown to users. This is for example what happens in the [WordPress Playground](https://playground.wordpress.net/) – it is vanilla WordPress that defaults to SQLite.
- Similarly, if the server supports MySQL and not SQLite, then the installation will use MySQL and no option will be shown to users.
- The UI option will only show if both SQLite and MySQL are supported by the server.

How can this implementation be tested?

The next step would be to thoroughly test the implementation with all the plugins you normally use. You can use the [SQLite Database Integration plugin](https://w.org/plugins/sqlite-database-integration/) to test an SQLite database on your existing website, or better yet – you can test the [pull request in WordPress Core](https://github.com/WordPress/wordpress-develop/pull/3220).

### 🤖 ··· WP-CLI 2.8.0 ···

[WP-CLI v2.8.0 release date](https://make.wordpress.org/cli/2023/04/05/wp-cli-v2-8-0-release-date/)

The v2.8.0 release of WP-CLI is scheduled to be published on 2023-05-03.

Among others a multitude of bug fixes and some new features, we’re also expecting this release to fix all known PHP 8.2 issues within WP-CLI itself.

The release candidate for this release will already be available sometime during the week prior, to allow for early testing. We will announce the availability of that release candidate in the #hosting-community channel to allow hosters to do early smoke testing.

### 🗓️ ···  Hosting Contributor Day ···

Heads up, some events are coming down the pipeline along with some Contributor Days to participate in:

- 2023-05-05, [WordCamp Barcelona](https://barcelona.wordcamp.org/2023/), Barcelona (Spain).
- 2023-06-08, [WordCamp Europe](https://europe.wordcamp.org/2023/), Athens (Greece).
- 2023-08-24, [WordCamp US](https://us.wordcamp.org/2023/), National Harbor, Maryland (USA).

Is anyone [going to a WordCamp](https://central.wordcamp.org/schedule/) and wants to lead a Hosting Table?

-----

Great, so just a look at our projects.

### 📕 ··· Advanced Administration Handbook ···

The Advanced Administration handbook is now public and online.

- [Public Handbook](https://developer.wordpress.org/advanced-administration/)
- [GitHub Handbook](https://github.com/WordPress/Advanced-administration-handbook)

Currently, [everything pending to do is in the Issues part](https://github.com/WordPress/Advanced-administration-handbook/issues). A lot of "documentation + enhancement" tickets, and also some "documentation + good first issue" for Contributor Day.

If anybody needs help or mentoring for this project, @javiercasares is always available.

### :cloud: ··· Host Test Runners ···

You may have heard of [Host Test Results](https://make.wordpress.org/hosting/test-results/), a community-led project that helps WordPress users find the best hosts for their sites. Host Test Results provides a comprehensive list of hosting providers and detailed information about each host's performance and reliability.

We want to make sure that as many hosts as possible are included in the Host Test Results database, so that WordPress users, the #core-test, and the #core team can benefit from this valuable resource. That's why we're asking you to check if your host is already supporting Host Test Results, and to add your host if it still needs to be included.

To check if your hoster is already supporting Host Test Results, visit the website (https://make.wordpress.org/hosting/test-results/) and search for your host's name. If your host is already included, great! You can use the information provided to make sure that you're getting the best possible performance and reliability from your site. 

If your host isn't included, don't worry, you can still help! You can just follow the instructions on the website to add your host to the database. It's quick and easy, and you'll be helping other WordPress users find the best hosts for their sites. However, if your host has not received reports in the last 25 revisions, we encourage you to check back with them.

- [Try the PHPUnit Test Runner](https://make.wordpress.org/hosting/handbook/tests/#try-the-phpunit-test-runner)
- [How to report: Creating your bot for WordPress.org](https://make.wordpress.org/hosting/handbook/tests/#how-to-report-creating-your-bot-for-wordpress-org)

-----

## Open Floor

Ok, if nothing else, let’s move on to 🤖 Open Floor. Folks are welcome to bring any topic up for discussion at this time.

-----

## Next meeting

⏰ Here are the [team's upcoming meetings](https://make.wordpress.org/meetings/#hosting):

- 2023-04-26 at 18:00 UTC (in a few hours)
- 2023-05-03 at 09:00 UTC (next week)
- 2023-05-03 at 18:00 UTC

📌 Okay, we're out of time for today, let's close out this meeting.

You are welcome to continue the conversation here in the #hosting-community channel or on Make/Hosting, at any time.

Thank you all for joining and for your contributions to WordPress!
See you at the next Hosting Team weekly meeting! < /meeting >

**Comment by Crixu** [comment]
Early meeting 5ftF Contributions:

- jameslhc
- crixu
- aristath
- JavierCasares
- mazzomaz
- chaion07
- monzuralam

**Reply by jessfrick** [reply]
Late meeting 5ftF Contributions:

- JavierCasares
- jameslhc
- jessibelle
